pyroute2/netlink/core.py

```diff
@@ -338,7 +356,9 @@ def event_loop(self):
                     message='Predefined event loop can not be used '
                     'in another thread',
                 )
-            self.local.event_loop = self.setup_event_loop()
+                self.local.event_loop = self.use_event_loop
```

Comment:
I think that your are fixing a bug here, but it has side effect in our codebase:

```
   File "routing.py", line 2668, in _get_primary_interface_mac_address
     ipr.close()
   File "pyroute2/pyroute2/netlink/core.py", line 804, in close
     self.asyncore.close(code)
   File "pyroute2/pyroute2/netlink/core.py", line 473, in close
    self.event_loop.close()
   File "/usr/lib/python3.11/asyncio/unix_events.py", line 68, in close
     super().close()
   File "/usr/lib/python3.11/asyncio/selector_events.py", line 88, in close
     raise RuntimeError("Cannot close a running event loop")
```

So I was not able to look deeper today, but I will try to test this MR without this change tomorrow

Reply:
Oh, that's a line from another fix, added here accidentally. And that's why it is breaking, the fix is not complete.

I removed the line, pushed over.

The issue that fixes this particular line will be addressed in the next PR for another bug.

Reply:
Thank you, I tried the MR and it fixes our problem, thank you! It looks a little bit slower, but it does not consume all the RAM anymore